en/docs/learn/design-api/publish-api/publish-to-multiple-external-api-stores.md

```diff
@@ -91,7 +91,7 @@ In this guide, let's use two separate instances of WSO2 API Manager, and let's p
             <!--Configuration to set the Developer Portal URL of the current running APIM deployment. 
             APIs published to external developer portals will be redirected to this URL-->
             
-                <StoreURL>http://<ip_address>:<port>localhost:9463/devportal</StoreURL>
+                <StoreURL>http://<ip_address>:<port>localhost:9444/devportal</StoreURL>
```

Comment:
The port should be 9443

Reply:
As per document it is referring the second instance with offset=1 and according to that port should be 9444.

Reply:
Also document says port should increment by ,1 according to it should be 9444 instead of 9463 for external devportal.

Reply:
@arunpatil1983 Ya the first too suggestions are correct. But I feel this one is referring to the main dev portal which should not have an offset. The offset applies to the ExternalAPIStore configuration which is mentioned just below this element

Reply:
Corrected at line no 94 .please review.

Reply:
Corrected at line no 94 .please review.

Reply:
Corrected at line no 94 .please review.